.github/workflows/nightly.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish Nightly Builds
+
+on:
+  push:
+    branches:
+      - nightly_build
+
+jobs:
+  build-test-publish:
+    strategy:
+      matrix:
+        include:
+          - runson: macos-latest # mac
+            archive_postfix: macos-arm64
+
+          - runson: ubuntu-latest # linux x64
+            image: docker://public.ecr.aws/async-profiler/asprof-builder-x86:latest
+            arch: x86_64
```

Comment:
`matrix.arch` is only used in musl-gcc path. If the path is the same for both architectures, the variable is not needed.

Reply:
am now using `/usr/local/musl` as the path